ducktape/src/test/scala/io/github/arainko/ducktape/fallible/accumulating/DerivedInstanceSuite.scala

```diff
@@ -75,10 +75,10 @@ class DerivedInstanceSuite extends DucktapeSuite {
 
       val expected =
         Left(
-          "Invalid Talk.Name" ::
-            "Invalid Talk.ElevatorPitch" ::
+          "Invalid Presenter.Bio" ::
```

Comment:
The new encoding results in the errors being reported in the inverse order - I might want to come back to this and make it so it keeps the order the same